app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt

```diff
@@ -167,14 +191,24 @@ fun BusScheduleScreen(
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(dimensionResource(R.dimen.padding_medium)),
+                .padding(dimensionResource(R.dimen.padding_medium))
```

Comment:
Can these two `.padding()` function calls be combined into one or one of them removed?